types/index.d.ts

```diff
@@ -222,21 +222,21 @@ declare namespace dayjs {
      * ```
      * dayjs().add(7, 'day')// => Dayjs
      * ```
-     * Units are case insensitive, and support plural and short forms.
+     * Units are case insensitive, support plural and short forms, and default to 'ms' if not supplied.
      *
      * Docs: https://day.js.org/docs/en/manipulate/add
      */
-    add(value: number, unit: OpUnitType): Dayjs
+    add(value: number, unit?: OpUnitType): Dayjs
     /**
      * Returns a cloned Day.js object with a specified amount of time subtracted.
      * ```
      * dayjs().subtract(7, 'year')// => Dayjs
      * ```
-     * Units are case insensitive, and support plural and short forms.
+     * Units are case insensitive, support plural and short forms, and default to 'ms' if not supplied.
```

Comment:
IMO it's better not to add this note, this will make our user confusing, and the code is hard to read.

Reply:
sounds good. I pushed a commit just now to revert those changes to those doc strings.